apps/frontend/src/lib/components/LanguageSwitcher.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+	import { locale, availableLocales } from '$lib/stores/locale';
```

Comment:
We have available locales available as `locales` in the Paraglide JS runtime. https://inlang.com/m/gerre34r/library-inlang-paraglideJs/runtime#locales